homework-submission/myapp4/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './Components/Header';
+import TodosCounter from "./Components/TodosCounter";
+import { inject, observer } from 'mobx-react';
+import Form from './Components/Form';
+
+
+@inject("TodosStore")
+@observer
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    props.TodosStore.addTodos();
+    
+  }
+  
+  deleteTodo = (id) => {
+    this.props.TodosStore.removeTodo(id);
+  };
+
+  isTaskDone = todo => {
+    this.props.TodosStore.isDone(todo);
+  };
+
+  renderTodos = () => {
+    if (this.props.TodosStore.Todos.data.length === 0)
+      return <p className="noItems">No items...</p>;
+
+    const textStyle = {
+      textDecoration: "line-through",
+      backgroundColor: "#0455d8"
+    };
+
+    return (
+      <ul>
+        {this.props.TodosStore.Todos.data.map((todo, id) => {
+          return <li key={id}>
```

Comment:
While we are receiving a one from the server I am not sure if we still need to use the index here, what do you say we have to use? 🙂